src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Core/Microsoft.Azure.KeyVault.Core.csproj

```diff
@@ -5,8 +5,8 @@
     <Description>Microsoft Azure Key Vault Core Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Core</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.Core</AssemblyName>
-    <VersionPrefix>2.0.4</VersionPrefix>
-    <PackageTags>Microsoft Azure key vault core</PackageTags>
+    <VersionPrefix>2.0.5-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure Key Vault Core</PackageTags>
```

Comment:
@schaabs can we add packageRelease Notes as below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10